include/boost/graph/isomorphism.hpp

```diff
@@ -366,7 +367,39 @@ namespace detail
 
             {
             return_point_true:
-                return true;
+                {
+                    std::vector< vertex1_t > unmatched_g1_vertices;
+                    BGL_FORALL_VERTICES_T(v, G1, Graph1)
+                    {
+                        if(f[v] == graph_traits< Graph2 >::null_vertex()) {
+                            unmatched_g1_vertices.push_back(v);
+                        }
+                    }
+
+                    if(!unmatched_g1_vertices.empty()) {
+                        typedef unordered_multimap< invar2_value, vertex2_t > g2_invar_vertex_multimap;
+                        typedef typename g2_invar_vertex_multimap::iterator multimap_iter;
+                        g2_invar_vertex_multimap unmatched_invar_multimap;
```

Comment:
Please try not to repeat the type in the variable name, so in this case please remove the `_multimap` suffix.

Also, `invar` is a confusing contraction -- just use the full word `invariant` in all cases.